cve_bin_tool/checkers/avahi.py

```diff
@@ -12,7 +12,12 @@
 
 
 class AvahiChecker(Checker):
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS = [
+        r"avahi_free",
+        r"avahi_strerror",
+        # r"avahi_string_list_free",
+        # r"libavahi-common.so.3",
```

Comment:
I'm curious about why these strings are commented out 🤔

Reply:
If you're going to put a few commented out strings in as backups, do put a comment explaining why they're there and where they came from. It can be pretty wordy and take up a few lines if you want, like "The strings below were found with <some sort of link to the helper script or helper script docs> as potential alternate contains patterns if the currently used strings (above) stop working in future versions."

Or you could be terse in the checker but link to a longer explanation elsewhere.
```
# Alternate optional contains patterns, see <link to helper script docs> for more details
```